client/src/components/existing-request/existing-request-display.vue

```diff
@@ -185,6 +186,12 @@ export default class ExistingRequestDisplay extends Vue {
       newReqModule.patchNameRequestsKV(payload)
     }
   }
+
+  /** Open Incorporate Now Login Modal and apply NR Data to Session */
+  activateILModal () {
+    newReqModule.mutateIncorporateLoginModalVisible(true)
+    sessionStorage.setItem('NR_DATA', JSON.stringify(this.nr))
```

Comment:
I finally found where this data gets used... but please add a comment here indicating this.

Reply:
Will do 👍